src/attributes.cpp

```diff
@@ -1264,6 +1264,7 @@ namespace attributes {
                 Rcpp::CharacterVector match = initMatches[i];
                 if (match.size() > 0) {
                     hasPackageInit_ = true;		// #nocov start
+                    showWarning(pkgInit + " found in C++ code so not generating RcppExports.cpp.");
```

Comment:
I am not a fan of missing changes to code into a PR which claimed at the outset to be "light and easy and docs only".